etc/profile-a-l/gnome-boxes.profile

```diff
@@ -0,0 +1,75 @@
+# Firejail profile for gnome-boxes
+# Description: Simple GNOME application to access virtual systems
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gnome-boxes.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/gnome-boxes
+noblacklist ${HOME}/.config/gnome-boxes
+noblacklist ${HOME}/.local/share/gnome-boxes
+noblacklist ${RUNUSER}/libvirt
+
+noblacklist /sbin
+noblacklist /usr/sbin
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+# breaks app
+#include disable-proc.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/gnome-boxes
+mkdir ${HOME}/.config/gnome-boxes
+mkdir ${HOME}/.local/share/gnome-boxes
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.cache/gnome-boxes
+whitelist ${HOME}/.config/gnome-boxes
+whitelist ${HOME}/.local/share/gnome-boxes
+whitelist ${RUNUSER}/libvirt
+
+whitelist /run/libvirt
+whitelist /usr/libexec/gnome-boxes*
+whitelist /usr/share/gnome-boxes
+whitelist /usr/share/libvirt
+whitelist /usr/share/osinfo
+whitelist /usr/share/qemu
+whitelist /usr/share/seabios
+whitelist /usr/share/vala*
+# /usr/share/misc/usb.ids is a symlink to /var/lib/usbutils/usb.ids on Ubuntu 22.04
+whitelist /var/lib/usbutils/usb.ids
```

Comment:
```suggestion
```

Is the last whitelisting needed?

Whitelisting follows symlinks and the relevant paths are already in
whitelist-usr-share-common.inc:

```
whitelist /usr/share/hwdata # Arch
whitelist /usr/share/misc # Ubuntu
```

```console
$ pacman -Qo /usr/share/hwdata/usb.ids
/usr/share/hwdata/usb.ids is owned by hwdata 0.379-1
$ pacman -Qlq hwdata
/usr/
/usr/share/
/usr/share/hwdata/
/usr/share/hwdata/iab.txt
/usr/share/hwdata/oui.txt
/usr/share/hwdata/pci.ids
/usr/share/hwdata/pnp.ids
/usr/share/hwdata/usb.ids
/usr/share/pkgconfig/
/usr/share/pkgconfig/hwdata.pc
```

Reply:
Same deal as with virt-manager. During installation of a VM (I used void linux) gnome-boxes refuses to install it and reports:

```sh
...
(gnome-boxes:42): Boxes-WARNING **: 01:38:34.659: os-database.vala:51: Error loading default libosinfo database: Error opening file /usr/share/misc/usb.ids: No such file or directory
...
```

Again, not sure if something else is broken related to whitelisting following symlinks.
I'll try to find some time to look into that. Could take afew days though, so I won't rush to get this in.

Thanks for the review!